pkg/apb/deprovision.go

```diff
@@ -6,7 +6,9 @@ import (
 )
 
 // Deprovision - runs the abp with the deprovision action.
-func Deprovision(instance *ServiceInstance, clusterConfig ClusterConfig, log *logging.Logger) (string, error) {
+func Deprovision(
+	instance *ServiceInstance, clusterConfig ClusterConfig, log *logging.Logger,
+) (string, error) {
```

Comment:
Thanks I don't mind the formatting in general but it's distracting when it is purely a format change.